hack/latest-csv-checksum.md5

```diff
@@ -1 +1 @@
-49a3d451b105a0f9b4d7f2b5ce0c2074
\ No newline at end of file
+9969e331faf9a9bf3dd36ed8969a12a3
```

Comment:
This shouldn't be needed. Try and remove it from the patch. (As @jarrpa requested.)

Reply:
Yup, done...